### PR TITLE
[ARRCONN-226] Trying to reconnect once after disconnection

### PR DESCRIPTION
**Pull request by vladimir-trifonov**
https://jira.appcelerator.org/browse/ARRCONN-226

### CODE REVIEW

File: lib/utility/_query.js [hunk 1]
```diff
@@ -50,19 +50,43 @@ exports._query = function _query(query, data, options, callback, executor) {
...
-
+	/**
+	 * Execute query over db.
+	 */
+	var executeQuery = function (connection, reconnect) {
 		/**
 		 * Row prefetching allows multiple rows to be returned from Oracle Database to Node.js in each
 		 * network round-trip. We can control how many rows are prefetched via the prefetchRows property
 		 * of either the base driver class or the options object passed to the execute method of a connection.
 		 */
```
Comment (janvennemann) [inline lib/utility/_query.js]: Please move that comment up to the if block starting in Line 42. It's not clear what that comment refers to now that the prefetch function if further down.

**Comment by vladimir-trifonov** [comment]
Yes, you're right about the comment. It's fixed!

**Comment by janvennemann** [comment]
Looks good! Please bump the version and i'll merge

**Comment by vladimir-trifonov** [comment]
@janvennemann Version bumped :)

**Pull request merged**